src/mono/netcore/System.Private.CoreLib/src/System/Threading/ThreadPool.Mono.cs

```diff
@@ -32,6 +33,7 @@ public static bool BindHandle(IntPtr osHandle)
             throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // Replaced by ThreadPoolBoundHandle.BindHandle
         }
 
+        [UnsupportedOSPlatform("browser")]
         public static bool BindHandle(SafeHandle osHandle)
```

Comment:
I think this needs to be added to https://github.com/dotnet/runtime/blob/3c1b48ee29f13cac93a86829374b917375a49a2d/src/coreclr/src/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs#L355 as well